net/aircrack-ng/Makefile

```diff
@@ -36,6 +38,19 @@ define Package/aircrack-ng/description
   WLAN tools for breaking 802.11 WEP/WPA keys
 endef
 
+define Package/airmon-ng
+  SECTION:=net
+  CATEGORY:=Network
+  DEPENDS:=+wireless-tools +ethtool +procps +usbutils +pciutils
```

Comment:
usbutils and pciutils are also both optional, but I would lean towards keeping pciutils as a hard dep.  maybe there is a way to dep on usbutils for platforms that have usb but not for everyone?